core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -120,11 +120,48 @@ var SkillEditorPage = function() {
     });
   };
 
-  this.expectRubricExplanationToMatch = function(index, explanation) {
-    rubricExplanations.then(function(explanations) {
-      explanations[index].getText().then(function(text) {
-        expect(text).toMatch(explanation);
-      });
+  this.deleteRubricExplanationWithIndex = function(diffIndex, explIndex) {
```

Comment:
ditto: avoid indexing if possible

Reply:
Done, used difficulty name everywhere, but explanation is still referenced using indices.